ui/hooks/src/moderation-requests.ts

```diff
@@ -110,7 +110,11 @@ export interface ModeratedItemsReponse extends PaginatedResponse {
 
 /**
  * Creates moderation entry
- * @returns - status of the response
+ * @returns response HTTP status code
+ * @example Creating a moderation entry
+ * ```typescript
+ * const status = await createModeration('https://someendpoint', {contentId: 'jdiupjkhfjh', contentType: 'post', data: object, signature: 'sig'}, 12000);
```

Comment:
instead of `jdiupjkhfjh` can we use something a bit more meaningful? like `some-content-id`

Reply:
Sure. I'll apply thise fixes